src/main/webapp/app/tutor-course-dashboard/tutor-course-dashboard.component.html

```diff
@@ -102,6 +102,10 @@ <h2>
                             [tutorParticipation]="exercise.tutorParticipations[0]"
                             [numberOfParticipations]="exercise.numberOfParticipations"
                             [numberOfAssessments]="exercise.numberOfAssessments"
+                            [numberOfComplaints]="exercise.numberOfComplaints"
```

Comment:
I would argue that we can just remove all these inputs from the component since it already has the whole exercise as an input

Reply:
I think the reason for this is that `jhi-tutor-participation-graph` is used in a different way in `tutor-exercise-dashboard.component.html` where those numbers are not taken from the exercise but are retrieved separately from the `/api/exercises/{exerciseId}/stats-for-tutor-dashboard` endpoint.